assets/pages/contributions-landing/helpers/context.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { getQueryParameter } from 'helpers/url';
+import { parseBoolean } from 'helpers/utilities';
+
+import { setContext } from '../actions/contributionsLandingActions';
+import type { Action } from '../actions/contributionsLandingActions';
+
+
+// ----- Functions ----- //
+
+function saveContext(dispatch: Action => void) {
+
+  const context = getQueryParameter('context', 'false');
+
+  if (context) {
+    dispatch(setContext(parseBoolean(context, false)));
```

Comment:
I would put the parseBoolean outside the if so in that way you are sure that the context always exists.

Reply:
It defaults to `false` in the initial state.